SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -122,11 +173,28 @@ TArray<SpatialLoadBalanceEnforcer::AclWriteAuthorityRequest> SpatialLoadBalanceE
 
 		if (StaticComponentView->HasAuthority(Request.EntityId, SpatialConstants::ENTITY_ACL_COMPONENT_ID))
 		{
-			PendingRequests.Push(AclWriteAuthorityRequest{ Request.EntityId, *DestinationWorkerId });
+			EntityAcl* Acl = StaticComponentView->GetComponentData<EntityAcl>(Request.EntityId);
+			auto* HeartbeatSets = Acl->ComponentWriteAcl.Find(SpatialConstants::HEARTBEAT_COMPONENT_ID);
```

Comment:
```suggestion
			WorkerRequirementSet* HeartbeatSets = Acl->ComponentWriteAcl.Find(SpatialConstants::HEARTBEAT_COMPONENT_ID);
```

Reply:
Also move into the `if` statement

Reply:
no idea how this even worked before, I had assumed it was a list of requirement sets

Reply:
also done